### PR TITLE
add usage examples/doctests

### PR DESCRIPTION
**Pull request by WilliamHYZhang**
added usage example/doctests for tf.math.is_non_decreasing and tf.math.is_strictly_increasing

### CODE REVIEW

File: tensorflow/python/ops/check_ops.py [hunk 1]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
 
   See also:  `is_strictly_increasing`
 
+  ```python
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: I don't believe the starting or ending triple backticks are needed.

File: tensorflow/python/ops/check_ops.py [hunk 2]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
...
+  >>> x1 = tf.constant([1.0, 1.0, 3.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Can you please simply remove the tf.print so we see the output boolean tensor?

File: tensorflow/python/ops/check_ops.py [hunk 3]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
...
+  1
+
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Remove blank line?

File: tensorflow/python/ops/check_ops.py [hunk 4]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
...
+  >>> x2 = tf.constant([3.0, 1.0, 2.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
+  0
+
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Remove blank line?

File: tensorflow/python/ops/check_ops.py [hunk 5]
```diff
@@ -1960,6 +1971,17 @@ def is_strictly_increasing(x, name=None):
 
   See also:  `is_non_decreasing`
 
+  ```python
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Please apply all the same comments from tf.math. is_non_decreasing

File: tensorflow/python/ops/check_ops.py [hunk 6]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
...
+  >>> x1 = tf.constant([1.0, 1.0, 3.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
+  1
+
+  >>> x2 = tf.constant([3.0, 1.0, 2.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
+  0
+
+  ```
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Remove triple backticks

File: tensorflow/python/ops/check_ops.py [hunk 7]
```diff
@@ -1926,6 +1926,17 @@ def is_non_decreasing(x, name=None):
 
   See also:  `is_strictly_increasing`
 
+  ```python
+  >>> x1 = tf.constant([1.0, 1.0, 3.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
+  1
+
+  >>> x2 = tf.constant([3.0, 1.0, 2.0])
+  >>> tf.print(tf.math.is_non_decreasing(x1))
```
Comment (jaingaurav) [inline tensorflow/python/ops/check_ops.py]: Remove tf.print

**Review by jaingaurav — CHANGES_REQUESTED** [review]

**Comment by WilliamHYZhang** [comment]
Thanks for the review, sorry for the late reply. Pushed updates that should address the requested changes. Thanks!

**Review by jaingaurav — APPROVED** [review]

**Pull request merged**